imagetest/test_suites/security/image_security_test.go

```diff
@@ -271,12 +271,12 @@ func verifyAutomaticUpdate(image string) error {
 	if err != nil {
 		return err
 	}
-	switch {
-	case strings.Contains(image, "debian-9"):
+	if strings.Contains(image, "debian-9") {
```

Comment:
the switch statement was fine here, please restore. repeated exclusive cases are where switch statements shine.